src/python_testing/TC_RVCOPSTATE_2_1.py

```diff
@@ -176,51 +235,96 @@ async def test_TC_RVCOPSTATE_2_1(self):
 
             if self.check_pics("RVCOPSTATE.S.M.ERR_NO_ERROR"):
                 self.print_step("7a", "Manually put the device in the no error state")
-                input("Press Enter when done.\n")
+                if not self.is_ci:
+                    input("Press Enter when done.\n")
                 await self.read_and_validate_operror(step="7b", expected_error=Clusters.OperationalState.Enums.ErrorStateEnum.kNoError)
             if self.check_pics("RVCOPSTATE.S.M.ERR_UNABLE_TO_START_OR_RESUME"):
                 self.print_step("7c", "Manually put the device in the unable to start or resume error state")
-                input("Press Enter when done.\n")
+                if self.is_ci:
```

Comment:
This is repeated often enough that it's probably worth popping into a function.

Reply:
I spent some time trying to make a common function that can take a lambda to execute if the this is running in CI, else waits for user input. I could not find a way to do this because the lambda needs to be async so that it can call the async methods. Unfortunately, lambdas cannot be async. 

I also cannot make the methods that are called in theses functions non-async since there is no way to call an async function from non-async function, even if you use await when calling the async function.

If there is a way to do this please enlighten me. I feel like this should be possible somehow.

Reply:
I'm going to resolve this for now but if there is a way to do this I can open another PR.